<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stripper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Import .txt</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="importLocation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Location</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="exportLocation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
